BUILD.bazel

```diff
@@ -2898,3 +2899,10 @@ genrule(
     """,
     local = 1,
 )
+
+# Run `bazel run //:refresh_compile_commands` to generate the `compile_commands.json` file.
+refresh_compile_commands(
+    name = "refresh_compile_commands",
+    # targets should include all C++ code.
+    targets = ["//:ray_pkg"],
```

Comment:
I would think you would need to hit all the cpp rules? like `//cpp/...` maybe?

Reply:
There is no such a single rule that contains all cpp code, unfortunately. I can change this to the concrete list of rules as well.